.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+
+## Fixes Issue
```

Comment:
Can you remove this section @neutr0n420? It is unnecessary because it is too specific to include in all pull requests. A pull request can be made to add a feature, refactor, test, update the build etc.

Reply:
I do think we should add a heading section at the top don't you think the same?

Reply:
I agree there should be a heading section with the purpose of summarizing the changes in the pull request. This is what I meant by `What changed`. Is there other information that you think should go in another heading section?